### PR TITLE
Merged code

### PR DESCRIPTION
**Pull request by raghuvaran-cpu**

### CODE REVIEW

File: app/Components/RegisterUser/RegisterUser.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ class Registration extends React.Component {
                     console.log(res.status)
                     if (res.status === 200) {
                         this.props.submitRegister(res.data.data)
-                        this.props.navigation.navigate('chatscreen');
+                        this.props.navigation.navigate('appscreen');
```
Comment (DivyaAndavarapu264) [inline app/Components/RegisterUser/RegisterUser.js:83]: Use the names as camelCase.

File: app/Components/chatRoom/chatRoom.js [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+
+  onMessages = data => {
+    this.setState({messages: data.messages});
+    this.socket.off('messages', true);
+  };
+
+  sendTypingStartStatus = () => {
+    this.socket.emit('typing-start', {
+      username: this.props.user.username,
+      client2: this.props.client.username,
```
Comment (DivyaAndavarapu264) [inline app/Components/chatRoom/chatRoom.js:75]: Use the proper naming convention dont use 1 or 2 like that

File: app/Components/chatRoom/chatRoom.js [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+
+  sendTypingEndStatus = () => {
+    this.socket.emit('typing-end', {
+      username: this.props.user.username,
+      client2: this.props.client.username,
+    });
+  };
+
+  send = () => {
+      console.log('send');
```
Comment (DivyaAndavarapu264) [inline app/Components/chatRoom/chatRoom.js:87]: Remove unnecessary console logs

File: app/Components/chatRoom/chatRoom.js [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+    if (this.state.chatMessage) {
+      this.socket.emit('chat', {
+        username: this.props.user.username,
+        client2: this.props.client.username,
+        message: this.state.chatMessage,
+      });
+      this.setState({chatMessage: ""});
```
Comment (DivyaAndavarapu264) [inline app/Components/chatRoom/chatRoom.js:94]: Give a comment there why you are doing empty to that chatMessage variable.Try to write comments Before starting the functions

File: app/Components/login/login.js [hunk 5]
```diff
@@ -115,7 +115,7 @@ class Login extends Component {
           console.log(res.data);
           if (res.status === 200) {
             this.props.userLogin(res.data.data);
-            this.props.navigation.navigate('chatscreen');
+            this.props.navigation.navigate('appscreen');
```
Comment (anurag-chitti) [inline app/Components/login/login.js:118]: Naming conventions should be camel case.

File: app/Components/chatScreen/chatScreen.js [hunk 6]
```diff
@@ -198,6 +247,12 @@ class ChatScreen extends Component {
     });
   }
 
+  onConversationClick = (user) => {
+    console.log("user");
```
Comment (anurag-chitti) [inline app/Components/chatScreen/chatScreen.js:251]: remove console

File: app/Components/chatScreen/chatScreen.js [hunk 7]
```diff
@@ -158,7 +182,9 @@ class ChatScreen extends Component {
         },
       })
       .then(res => {
+        console.log('res',res.status);
```
Comment (anurag-chitti) [inline app/Components/chatScreen/chatScreen.js:185]: remove console

File: app/Components/chatScreen/chatScreen.js [hunk 8]
```diff
@@ -158,7 +182,9 @@ class ChatScreen extends Component {
...
         if (res.status === 200) {
+          console.log('status 200',res.status);
```
Comment (anurag-chitti) [inline app/Components/chatScreen/chatScreen.js:187]: remove console

File: app/Components/Routing.js [hunk 9]
```diff
@@ -1,36 +1,61 @@
...
-          <Stack.Screen name="chatscreen" component={ChatScreen} options={{ header: () => null }} />
-          <Stack.Screen name="contacts" component={Contacts} options={{ header: () => null }} />
-        </Stack.Navigator>
-      </NavigationContainer>
-    )
+      <NavigationContainer>
+      <TabNavigator.Navigator>
+        <TabNavigator.Screen name="chatscreen" component={ChatScreen} options={{ header: () => null }} />
+        <TabNavigator.Screen name="contacts" component={Contacts} options={{ header: () => null }} />
+        <TabNavigator.Screen name="chatroom" component={ChatRoom} options={{ header: () => null }} />
```
Comment (DivyaAndavarapu264) [inline app/Components/Routing.js:22]: Names should be in camelcase chatRoom and chatScreen should be changed

File: app/Components/chatRoom/chatRoom.js [hunk 10]
```diff
@@ -0,0 +1,430 @@
...
+    color: 'white',
+  },
+  options: {
+    color: 'white',
+    fontSize: 16,
+  },
+});
+
+const mapStateToProps = state => (
+  console.log('map state to props', state),
```
Comment (anurag-chitti) [inline app/Components/chatRoom/chatRoom.js:422]: remove console

File: app/Components/chatRoom/chatRoom.js [hunk 11]
```diff
@@ -0,0 +1,430 @@
...
+    onShowEmoji = () => {
+        this.setState({ showEmoji: !this.state.showEmoji });
+    }
+
+    addEmoji = (emoji) => {
+        this.setState({ message: this.state.message + emoji })
+    }
+
+    handleText = (msg) => {
+        console.log(msg);
```
Comment (anurag-chitti) [inline app/Components/chatRoom/chatRoom.js:107]: remove console

**Review by anurag-chitti — CHANGES_REQUESTED** [review]

**Pull request merged**